test/server/routes/users.test.js

```diff
@@ -0,0 +1,40 @@
+import request from 'supertest'
+import getToken from './get-token'
+
+jest.mock('../../../server/db/users', () => ({
+  getAllUsers: () => Promise.resolve([
+    {id: 1, ghid: '12345', username: 'test', hash: null, is_approved: true},
+    {id: 2, ghid: '125555', username: 'foo', hash: null, is_approved: false}
+  ]),
+  getUserById: () => Promise.resolve(
+    {id: 1, ghid: '12345', username: 'test', hash: null, is_approved: true}
+  )
+}))
+
+jest.mock('../../../server/routes/github', () => () => {})
+
+// eslint-disable-next-line
```

Comment:
Please add the rule you're disabling at the end of the eslint comment. If you remove the comment, VS Code should remind you which rule you're disabling.